src/focus.service.ts

```diff
@@ -472,8 +480,12 @@ export class FocusService {
       this.selectNode(ev.next, scrollSpeed);
       return true;
     } else if (ev.event === Direction.SUBMIT) {
-      this.selected.click();
-      return true;
+      if (this.selected) {
+        this.selected.click();
+        return true;
+      } else {
+        throw new Error('No node selected');
```

Comment:
This seems a little weird since elsewhere we're OK with selecting being null. We should probably either type selected as non-nullable if it shouldn't be null, or not throw here to avoid crashing the app if a user hits submit without an item being selected.